students/ak-foster/Spring/Lesson02/featuresdf.csv

```diff
@@ -0,0 +1,101 @@
+id,name,artists,danceability,energy,key,loudness,mode,speechiness,acousticness,instrumentalness,liveness,valence,tempo,duration_ms,time_signature
```

Comment:
try using .gitignore and learning to not push your csv files to github. That's my opinion, but other might not agree.